mlperf.conf

```diff
@@ -28,15 +30,17 @@ stable-diffusion-xl.*.performance_sample_count_override = 5000
 
 *.SingleStream.target_latency_percentile = 90
 *.SingleStream.min_duration = 600000
-#*.SingleStream.min_query_count = 1024
 
 *.MultiStream.target_latency_percentile = 99
 *.MultiStream.samples_per_query = 8
 *.MultiStream.min_duration = 600000
-#*.MultiStream.min_query_count = 270336
 *.MultiStream.min_query_count = 662
 retinanet.MultiStream.target_latency = 528
 
+# With early stopping enabled, the min_query_count is 1 for all non-Offline scenario
+*.Server.min_query_count = 1
+*.SingleStream.min_query_count = 1
```

Comment:
Even with EarlyStopping SingleStream needs a minimum of 100 queries and MultiStream needs a minimum of 662 queries.

Reply:
Let me fix that. Apology

Reply:
I have removed these lines and use default value for min_query_count